Lib/threading.py

```diff
@@ -1146,15 +1164,47 @@ def daemon(self, daemonic):
         self._daemonic = daemonic
 
     def isDaemon(self):
+        """Return whether this thread is a daemon.
+
+        This method is deprecated, use the .daemon property instead.
+
+        """
+        import warnings
+        warnings.warn('isDaemon() is deprecated, use .daemon instead',
+                      DeprecationWarning, stacklevel=2)
         return self.daemon
 
     def setDaemon(self, daemonic):
+        """Set whether this thread is a daemon.
+
+        This method is deprecated, use the .daemon property instead.
+
+        """
+        import warnings
+        warnings.warn('setDaemon() is deprecated, use .daemon instead',
```

Comment:
```suggestion
        warnings.warn('setDaemon() is deprecated, set the daemon attribute instead',
```

Same remark for other deprecated getter methods.